### DISCUSSION TITLE
[artalk] comment bar out of main menu

### DISCUSSION THREAD

**Discussion by musnows** [requarks/wiki#6015] [Error / Bug Report]
I encountered this problem. When I use the artalk comment system, the interface of the comment system will leave its place

<img width="869" alt="Snipaste_2023-01-05_19-45-21" src="https://user-images.githubusercontent.com/87506951/210774106-55991548-f27f-49f1-bfe5-dba1623c91e0.png">

I refreshed the webpage many times and changed the browser, but the results were the same (Edge & firefox)

> wiki-js version `2.5.292`

**Comment by musnows** [comment]
this issue still exites in `Wiki.js 2.5.295`

**Comment by musnows** [comment]
this issue still exites in `Wiki.js 2.5.297`, in edge & firefox

```
<div class="atk-main-editor">
```

## buttons  are hided behind comment bar

<img width="135" alt="Snipaste_2023-04-23_22-41-21" src="https://user-images.githubusercontent.com/87506951/233846365-718b1f8a-777f-4344-8bee-b0a3538a3cde.png">

## comment bar out of main menu

![image](https://user-images.githubusercontent.com/87506951/233846432-ac78c637-e298-4083-941c-42191a398cd0.png)

**Comment by cdn0x12** [comment]
Recent wiki.js updates didn't change the code of Artalk module.

The direct cause of this problem is that Artalk sets the `z-index` value of `#atk-main-editor` to `5` in `Artalk.css`. You need to change it to a non-negative integer less than 5 or `auto`.

It seems that Artalk currently does not provide an option to modify CSS styles upon importing, so the problem can't be fixed well from wiki.js' side. If you find maintaining CSS too troublesome, you can try limiting the height of comment from the backend to alleviate this issue.
